settings/icu_tokenizer.yaml

```diff
@@ -1,5 +1,13 @@
 query-preprocessing:
     - step: split_japanese_phrases
+    - step: regex_replace
```

Comment:
As said in the issue, please don't add anything here in the default config. You can use the IPv4/6 example in the tests and in the [documentation](https://github.com/osm-search/Nominatim/blob/master/docs/customize/Tokenizers.md#query-preprocessing) that still needs to be written.

Reply:
The code is still here.